packages/starknet-snap/src/utils/fee.ts

```diff
@@ -55,12 +55,19 @@ export class ConsolidateFees {
           fee.resourceBounds.l1_gas.max_price_per_unit,
         );
         acc.resourceBounds.l2_gas.max_amount += BigInt(
-          fee.resourceBounds.l2_gas.max_amount,
+          fee.resourceBounds.l2_gas.max_amount ?? BigInt(0),
```

Comment:
```suggestion
          fee.resourceBounds.l2_gas.max_amount ?? 0,
```

you dont need double bigInt? apply to same for all similar place